corehq/apps/case_search/templates/case_search/csql_expression_form.html

```diff
@@ -0,0 +1,33 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<form hx-post="{{ request.path_info }}"
+      hq-hx-action="save_expression"
+      hx-swap="outerHTML"
+      x-data="{ isUnsaved: false }"
+      @change="isUnsaved = true;"
+      @submit="isUnsaved = false;">
+  {% if form.instance.pk %}
+    <input type="hidden" name="pk" value="{{ form.instance.pk }}" />
+  {% endif %}
+  <div class="row pb-3">
+    <div class="col-2">
+      {{ form.name }}
+    </div>
+    <div class="col-8">
+      {{ form.csql }}
+    </div>
+    <div class="col-2">
+      <button type="submit" class="btn btn-outline-primary" x-bind:disabled="!isUnsaved">
```

Comment:
```suggestion
      <button
        class="btn btn-outline-primary" type="submit"
        :disabled="!isUnsaved"
      >
```
1) I like to always have `class` be the first attribute
2) if there is a `HTMX` or `alpine` attribute (`:bind` in this case), we want it on a second line
3) I prefer shorthand `alpine` bindings, but could be convinced otherwise?